graphql_compiler/cost_estimation/cardinality_estimator.py

```diff
@@ -69,17 +69,99 @@ def _get_last_edge_direction_and_name_to_location(location):
     return edge_direction, edge_name
 
 
-def _get_parent_and_child_name_from_edge(schema_graph, child_location):
+def _get_parent_and_child_base_classnames_from_edge(schema_graph, child_location):
```

Comment:
nit:`_get_base_parent_and_child_class_names_from_edge` reads better to me.

Reply:
Rewrote it as `_get_base_classnames_of_parent_and_child_from_edge`. Let me know what you think.